app/jfrog-logs/default/data/ui/views/xray.xml

```diff
@@ -312,9 +312,9 @@
   <row id="vulnerability6">
     <panel>
       <chart>
-        <title>Top Vulnerable Artifacts by Downloads (Based on Username)</title>
+        <title>Top Impacted Artifact by Count of User Downloads</title>
         <search>
-          <query>index="main" $vulnerability_control$ log_source="jfrog.rt.artifactory.request" return_status=200 [search index=xray_violations impacted_artifacts{}=* | stats count by impacted_artifacts{}  | rex field=impacted_artifacts{} "default\/(?&lt;rex_repo_path&gt;.*)" | return 500000 $rex_repo_path] | stats count(username) by request_url | rename request_url as impacted_artifact</query>
+          <query>index="jfrog_splunk" $vulnerability_control$ log_source="jfrog.rt.artifactory.request" return_status=200 [search index=xray_violations impacted_artifacts_url{}=* | stats count by impacted_artifacts_url{} | rex field=impacted_artifacts_url{} "(?&lt;impacted_artifacts_url&gt;.*)" | return 500000 $impacted_artifacts_url ] | stats count(username) by request_url | rename request_url as impacted_artifact</query>
```

Comment:
Can we use the same macro as the artifactory dashboard instead of hard coding the index?

Reply:
yes